HandleNHCreateOrUpdateInstallationCallOrchestrator/index.ts

```diff
@@ -0,0 +1,29 @@
+﻿import * as df from "durable-functions";
+import * as o from "../utils/durable/orchestrators";
+
+import { getConfigOrThrow } from "../utils/config";
+import { getHandler } from "./handler";
+
+import {
+  ActivityBodyImpl as CreateOrUpdateActivityBodyImpl,
+  activityName as CreateOrUpdateActivityName,
+  ActivityResultSuccess as CreateOrUpdateActivityResultSuccess
+} from "../HandleNHCreateOrUpdateInstallationCallActivity";
+import { getNHLegacyConfig } from "../utils/notificationhubServicePartition";
+
+const config = getConfigOrThrow();
+
+const createOrUpdateActivity = o.callableActivity<
+  CreateOrUpdateActivityBodyImpl
+>(CreateOrUpdateActivityName, CreateOrUpdateActivityResultSuccess, {
+  ...new df.RetryOptions(5000, config.RETRY_ATTEMPT_NUMBER),
+  backoffCoefficient: 1.5
+});
```

Comment:
Opinionated choice: activities referenced by the orchestrator are _built_ in the index file and passed to the business logic as a dependency.
`o.callableActivity` is a builder functions that wraps the Durable Functions' client to call a given activity handling its results.